monitoring/statsd/metrics.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from datadog import DogStatsd
+
+
+statsd_host = os.environ.get("STATSD_ENDPOINT")
+statsd_port = int(os.environ.get("STATSD_PORT"))
+statsd = DogStatsd(host=statsd_host, port=statsd_port)
+
+REQUEST_LATENCY_METRIC_NAME = "django_request_latency_seconds"
+REQUEST_COUNT_METRIC_NAME = "django_request_count"
+NUM_SUBMISSIONS_IN_QUEUE = "num_submissions_in_queue"
+NUM_PROCESSED_SUBMISSIONS = "num_processed_submissions"
+
+
+def set_statsd_gauge(metric_name, tags, set_value):
+    statsd.gauge(metric_name, set_value, tags=tags)
+
+
+def increment_statsd_counter(metric_name, tags, inc_value):
+    statsd.increment(metric_name, inc_value, tags=tags)
```

Comment:
Values will be incremented by 1. Why do we have to pass increment value explicitly?

Reply:
@Ram81 I didn't hardcore the value here to one as I thought it would be good to have a generic util function in which we can pass a custom increment value. Otherwise we can hardcore the increment value to one in the util function

Reply:
Prometheus counters have an option to increment any counter by any value and not just one, moreover it's a statsd metrics here which is converted to prometheus library by datadog